agent/acs/handler/attach_eni_handler.go

```diff
@@ -154,11 +157,11 @@ type ackTimeoutHandler struct {
 func (handler *ackTimeoutHandler) handle() {
 	eniAttachment, ok := handler.state.ENIByMac(handler.mac)
 	if !ok {
-		seelog.Warnf("Timed out waiting for ENI ack; ignoring unmanaged ENI attachment with MAC address:%s", handler.mac)
+		seelog.Warnf("Timed out waiting for ENI ack; ignoring unmanaged ENI attachment with MAC address: %s", handler.mac)
 		return
 	}
 	if !eniAttachment.IsSent() {
-		seelog.Info("Timed out waiting for ENI ack; removing ENI attachment record with MAC address: %s", handler.mac)
+		seelog.Infof("Timed out waiting for ENI ack; removing ENI attachment record with MAC address: %s", handler.mac)
 		handler.state.RemoveENIAttachment(handler.mac)
```

Comment:
I can see there may be race condition here in edge case where a eni attachment event was sent to eventhandler but before submitstatechange return, at this time the timer was triggered. This should be handled in another PR, just make sure we don't forget.